_posts/2024-03-19-FOSSRIT-bug-fix.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+
+# Uncomment only one of the below categories
+categories: 
+- Bug Fix
+#- Contribution
+
+
+# Enter your name below
+author: Joshua Talbot
+---
+
+## Introduction to the community
+
+The project I chose was [FOSSRIT website](https://fossrit.github.io/), which Adrien introduced me to during FOSS Hours early in the semester. I initially was looking at contributing to Godot, however getting used to the file structure of godot proved to be difficult.
```

Comment:
*Adri**a**n lol, I probably should have corrected you sooner rather than just silently editing the spelling in the "out your bug fix here" google doc